contracts/bprotocol/Registry.sol

```diff
@@ -18,7 +18,11 @@ contract Registry is Ownable {
     address public pool;
     address public bComptroller;
     address public score;
+
+    // Avatar
     address public avatarMaster;
+    bytes private avatarProxyContractCode;
```

Comment:
do we really need it?
maybe just make it constant. it will save storage read in deployment.